examples/job/create_and_submit_job.py

```diff
@@ -90,16 +100,10 @@
 
 
 config = {
-    "owner": {
-        "_id": owner_id
-    },
-    "_material": {
-        "_id": material_id
-    },
-    "workflow": {
-        "_id": workflow_id
-    },
-    "name": JOB_NAME
+    "owner": {"_id": owner_id},
```

Comment:
Let's configure the autoformatter to put newline after `{`

Reply:
Unfortunately, I don't think its possible to do this using the Black code formatter